src/qt/votingdialog.cpp

```diff
@@ -288,8 +288,8 @@ void VotingTableModel::resetData(bool history)
             item->question_ = QString::fromStdString(sQuestion);
             item->answers_ = QString::fromStdString(sAnswers);
             item->arrayOfAnswers_ = QString::fromStdString(sArrayOfAnswers);
-            item->totalParticipants_ = QString::fromStdString(sTotalParticipants);
-            item->totalShares_ = QString::fromStdString(sTotalShares);
+            item->totalParticipants_ = std::stoul(sTotalParticipants);
+            item->totalShares_ = std::stoul(sTotalShares);
```

Comment:
Could this raise exception on remotely injected invalid data?

Reply:
Should stop at first non-numerical value:

> Discards any whitespace characters (as identified by calling isspace()) until the first non-whitespace character is found, then takes as many characters as possible to form a valid base-n (where n=base) unsigned integer number representation and converts them to an integer value.